### DISCUSSION TITLE
Can't save specific page using the extension

### DISCUSSION THREAD

**Discussion by tiwalayo** [gildas-lormeau/SingleFile#1560] [Q&A]
**Describe the bug**

When I try to download [this site](https://anduril.com) using SingleFile with default settings, the navbar elements do not generate a dialog on hover as they do on the original page. When I follow the directions in the FAQ ("you can save [scripts] by unchecking the option "Network > blocked resources > scripts", and optionally unchecking "HTML Content > remove hidden elements", unchecking "Stylesheets > remove unused styles", and checking the option "HTML content > save raw page"), then the navbar elements load, but then the page either cannot scroll or does not generate the rest of the page (it's unclear which of these is going on, but the experience is as if any of them were).

**To Reproduce**
Go to [anduril dot com](https://anduril.com) and run SingleFile 1.22.65 on Chrome

**Expected behavior**

Faithful mirror of the main site (minus the mp4 on the front page) that replicates the page and is interactively functional with internet turned off

**Screenshots**

Default settings issue is self explanatory.

The "not generating" / "not scrolling" case looks like this (see how the menu with "Command and Control" shows up); however it is as if scrolling is disabled (though of course you can't tell from the screenshot)
<img width="1698" alt="image" src="https://github.com/user-attachments/assets/22cceded-989d-4888-80ad-54feb955e14d">

**Environment**
 - OS: MacOS Sonoma 14.
 - Browser: Chrome
 - Version: 64-bit

**Additional context**

**Comment by gildas-lormeau** [comment]
This webpage load some scripts dynamically. That's the reason why SingleFile cannot save it properly. Unfortunately, this is one of SingleFile's limitations.
